packages/fiori/src/Bar.js

```diff
@@ -133,6 +133,34 @@ class Bar extends UI5Element {
 			"label": this.design,
 		};
 	}
+
+	resizeHandler(entry) {
+		const bar = entry[0].target;
+		let changeFlex = false;
+		const barWidth = bar.offsetWidth;
+
+		for (let i = 0; i < bar.childElementCount; i++) {
```

Comment:
This loop is redundant. Only the last iteration will have effect. You could simplify this and remove the loop completely.

Reply:
The loop will give an answer if ANY of the children is larger. But i guess i can use the array.some method. It will be more clear.